File: KantaPe_Web/Kruma.KantaPe/Kruma.KantaPe.Entidad/SolicitudCancion.cs
using System;
using System.Runtime.Serialization;

namespace Kruma.KantaPe.Entidad
{
    /// <summary>SolicitudCancion</summary>
    /// <remarks><list type="bullet">
    /// <item><CreadoPor>Creado por Carlos Gómez</CreadoPor></item>
    /// <item><FecCrea>24-08-2016</FecCrea></item></list></remarks>

    [Serializable]
    [DataContract]
    public class SolicitudCancion
    {
        [DataMember]
        public int? IdSolicitudCancion { get; set; }
        [DataMember]
        public string Cancion { get; set; }
        [DataMember]
        public string Artista { get; set; }
        [DataMember]
        public string Album { get; set; }
        [DataMember]
        public string Idioma { get; set; }
        [DataMember]
        public string Genero { get; set; }
        [DataMember]
        public string Youtube { get; set; }
        [DataMember]
        public string Estado { get; set; }
        [DataMember]
        public string UsuarioCreacion { get; set; }
        [DataMember]
        public DateTime? FechaCreacion { get; set; }
        [DataMember]
        public string UsuarioModificacion { get; set; }
        [DataMember]
        public DateTime? FechaModificacion { get; set; }
        [DataMember]
        public string Motivo { get; set; }
        [DataMember]
        public string NombreCompleto { get; set; }
        [DataMember]
        public int? Cantidad { get; set; }
    }
}

File: app/src/main/java/com/kruma/kanta/entidad/SolicitudCancion.java
package com.kruma.kanta.entidad;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

/**
 * Created by Enrique on 25/08/2016.
 */
public class SolicitudCancion {

    //region ATRIBUTOS
    @SerializedName("Cancion")
    private String cancion;
    @SerializedName("Artista")
    private String artista;
    @SerializedName("Album")
    private String album;
    @SerializedName("Idioma")
    private String idioma;
    @SerializedName("Genero")
    private String genero;
    @SerializedName("Youtube")
    private String youtube;
    @SerializedName("Estado")
    private String estado;
    @SerializedName("UsuarioCreacion")
    private String usuarioCreacion;
    @SerializedName("FechaCreacion")
    private Date fechaCreacion;
    @SerializedName("UsuarioModificacion")
    private String usuarioModificacion;
    @SerializedName("FechaModificacion")
    private Date fechaModificacion;
    //endregion

    //region PROPIEDADES
    public String getCancion() {
        return cancion;
    }

    public void setCancion(String cancion) {
        this.cancion = cancion;
    }

    public String getArtista() {
        return artista;
    }

    public void setArtista(String artista) {
        this.artista = artista;
    }

    public String getAlbum() {
        return album;
    }

    public void setAlbum(String album) {
        this.album = album;
    }

    public String getIdioma() {
        return idioma;
    }

    public void setIdioma(String idioma) {
        this.idioma = idioma;
    }

    public String getGenero() {
        return genero;
    }

    public void setGenero(String genero) {
        this.genero = genero;
    }

    public String getYoutube() {
        return youtube;
    }

    public void setYoutube(String youtube) {
        this.youtube = youtube;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }

    public String getUsuarioCreacion() {
        return usuarioCreacion;
    }

    public void setUsuarioCreacion(String usuarioCreacion) {
        this.usuarioCreacion = usuarioCreacion;
    }

    public Date getFechaCreacion() {
        return fechaCreacion;
    }

    public void setFechaCreacion(Date fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }

    public String getUsuarioModificacion() {
        return usuarioModificacion;
    }

    public void setUsuarioModificacion(String usuarioModificacion) {
        this.usuarioModificacion = usuarioModificacion;
    }

    public Date getFechaModificacion() {
        return fechaModificacion;
    }

    public void setFechaModificacion(Date fechaModificacion) {
        this.fechaModificacion = fechaModificacion;
    }
    //endregion
}
